test-app/presenter/src/commonMain/kotlin/com/example/redwood/testing/presenter/TestApp.kt

```diff
@@ -26,12 +26,11 @@ import app.cash.paging.PagingState
 import app.cash.paging.compose.collectAsLazyPagingItems
 import app.cash.redwood.layout.api.Constraint
 import app.cash.redwood.lazylayout.compose.LazyColumn
-import app.cash.redwood.treehouse.TreehouseUi
 import kotlinx.serialization.json.Json
 
-class TestAppTreehouseUi(
+class TestApp(
   private val httpClient: HttpClient,
-) : TreehouseUi {
+) {
```

Comment:
Minimal changes to break the Treehouse dependency. Will refactor later as I add the concept of screens to this app.